File: EJERCICIOS KATTIS/Tarifa/Tarifa.cs
using System;
namespace ejercicios_kattis
{
    class Tarifa
    {
        static void Main(string[] args)
        {
            int x, n, p, a, b = 0, i;
            x = Convert.ToInt32(Console.ReadLine());
            n = Convert.ToInt32(Console.ReadLine());
            for (i = 0; i < n; i++)
            {
                p = Convert.ToInt32(Console.ReadLine());
                a = x - p;
                b = b + a; 
            }
            Console.WriteLine (b+x);
        }
    }
}

File: EJERCICIOS KATTIS/Tarifa/Tarifa.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Tarifa
{
    public static void main(String[] args) throws IOException
    {
        BufferedReader lc = new BufferedReader(new InputStreamReader(System.in));
        String xx, nn, pp;
        int x, n, p, a, b = 0;
        xx = lc.readLine();
        x = Integer.parseInt(xx);
        nn = lc.readLine();
        n = Integer.parseInt(nn);
        for(int i = 0; i < n; i++)
        {
            pp = lc.readLine();
            p = Integer.parseInt(pp);
            a = x-p;
            b = b + a;
        }
        System.out.print(b+x);
    }
}